src/main/scala/fixedpoint/FixedPoint.scala

```diff
@@ -326,10 +325,6 @@ sealed class FixedPoint private[fixedpoint] (width: Width, private var _inferred
 
   override def bulkConnect(that: Data)(implicit sourceInfo: SourceInfo): Unit = connectOp(that, _ <> _)
 
-  override def connectFromBits(that: Bits)(implicit sourceInfo: SourceInfo): Unit = {
-    this.data := that.asTypeOf(this.data)
-  }
-
```

Comment:
Noted by @tymcauley in Chisel dev meeting that this is the problem with upgrading to Chisel 7.

Reply:
Once https://github.com/chipsalliance/chisel/pull/4782 is merged and released, you should be able to fix this with:
```suggestion
  override protected def _fromUInt(that: UInt)(implicit sourceInfo: SourceInfo): Data = {                                                                                                                           
    val _w = Wire(this.cloneType)
    _w.data := that.asTypeOf(this.data)
    _w
  }

```